src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -351,6 +351,7 @@ private EditorServicesConfig CreateConfigObject()
             var profile = (PSObject)GetVariableValue("profile");
 
             var hostInfo = new HostInfo(HostName, HostProfileId, HostVersion);
+            var initialSessionState = Runspace.DefaultRunspace.InitialSessionState;
```

Comment:
Can you add a comment above this saying that we will use this initial session state that comes from the runspace that started Editor Services for all runspace creation in Editor Services?

Reply:
Done